content/en/docs/tutorials/kubernetes-basics/deploy-intro.html

```diff
@@ -89,7 +89,7 @@ <h2 style="color: #3771e3;">Deploying your first app on Kubernetes</h2>
 
         <div class="row">
             <div class="col-md-8">
-                <p>For our first Deployment, we'll use a <a href="https://nodejs.org">Node.js</a> application packaged in a Docker container. The source code and the Dockerfile are available in the <a href="https://github.com/kubernetes/website/tree/master/docs/tutorials/kubernetes-basics">GitHub repository</a> for the Kubernetes Basics.</p>
+                <p>For our first Deployment, we'll use a <a href="https://nodejs.org">Node.js</a> application packaged in a Docker container. The source code and the Dockerfile are available in the <a href="https://github.com/kubernetes/website/tree/master/content/en/docs/tutorials/kubernetes-basics">GitHub repository</a> for the Kubernetes Basics.</p>
```

Comment:
The update is correct as a link translation -- but it looks as though the actual location hasn't been valid for several releases now (checked 1.10 branch as well as master). There's no source code or Dockerfile at the specified location (old or new).

Because the actual tutorial starts outside this page, I recommend removing this section altogether. Adding others to the PR to review, though, in case they have better information.

Reply:
Indeed - thanks.